app/src/androidTest/java/com/sdp/movemeet/Navigation/MainActivityNavigationTest.java

```diff
@@ -121,35 +104,16 @@ public void mainActivityToStartActivity() {
 
     @Test
     public void mainActivityGotoHome() {
-        onView(withId(R.id.nav_view)).perform(NavigationViewActions.navigateTo(R.id.nav_start_activity));
-
-        /*Context context = ApplicationProvider.getApplicationContext();
-
-        Intent activity = new Intent(context, ActivityDescriptionActivity.class);
-        activity.putExtra(ActivityDescriptionActivity.EXTRA_ACTIVITY_ID ,DUMMY_ACTIVITY_ID);
-        activity.putExtra(String.valueOf(ActivityDescriptionActivity.EXTRA_LATITUDE),DUMMY_LATITUDE);
-        activity.putExtra(ActivityDescriptionActivity.EXTRA_DATE ,DUMMY_DATE);
-        activity.putExtra(ActivityDescriptionActivity.EXTRA_TITLE ,DUMMY_TITLE);
```

Comment:
Why did you remove this test?

Reply:
According to roxane's message just above, it is not necessary so I will delete it

Reply:
Ok